DOCS/license-inclusion-principles.md

```diff
@@ -0,0 +1,22 @@
+## License Inclusion Principles
```

Comment:
```suggestion
## License Inclusion Principles

_This document was copied from <https://spdx.org/spdx-license-list/license-list-overview> in early April of 2019. See that pages for the latest version._
```

Reply:
Hi @kemitchell, I think the goal is that we may begin iterating on the inclusion principles here in the repo, rather than on the website. So we'll likely merge this for now as-is, and may eventually move to this being the "latest version" / master copy if it evolves over time.